### ISSUE TITLE
Push notification taking too long to arrive 

### ISSUE THREAD

**Issue by Chileads** [BlueWallet/GroundControl#184]
Push notification can take up to 10hrs to arrive especially for incoming transactions. I mainly use the notification to know when their is incoming transactions because the transaction does not appear even when you open and refresh the app. When I get a notification I go to "coin control" to verify the amount. 

I learnt that it's cause of traffic in the groundcontrol-bluewallet.herokuapp.com server that's causing the delay in notification.
I suggest we should have multiple servers or alternatives for app notifications.

**Comment by Overtorment** [comment]
yes, the problem is the sender is a single process and is not refactored to run in multi-process.
sometimes send queue grows to tens of thousands and it takes hours to clear.

when the queue is not clogged notifications should be instant.

for some reason people add exchange's hot wallets as a watch only and receive all their their txs as pushes. some addresses have 300k+ transactions.

i did some work recently to ignore "thick" addresses to now slow things down for everyone else
